test/PwmRotationTest.cpp

```diff
@@ -0,0 +1,263 @@
+/**
+ * @file   PwmRotationTest.cpp
+ * @brief  PwmRotationクラスのテスト
+ * @author Negimarge
```

Comment:
```suggestion
 * @author takahashitom
```
これは合ってる？
もし根木原くんに手伝ってもらってても、担当分でファイルを扱ってるなら自分の名前書いていいよ🙆‍♀️

Reply:
すいません。コピペしてそのままでした。
修正しておきます。